package.json

```diff
@@ -18,7 +18,13 @@
   "homepage": "https://github.com/orn0t/uvs#readme",
   "dependencies": {
     "express": "^4.15.2",
-    "mongoose": "^4.9.9"
+    "mongoose": "^4.9.9",
+    "body-parser": "^1.15.2",
+    "cookie-parser": "^1.4.3",
+    "formidable": "^1.0.17",
+    "mongojs": "^2.4.0",
```

Comment:
Ok, lets use deps by `needs` not by `wants` or `futures`. Is any code usage for `mongojs` and `formidable` for now ?